media/js/mozorg/home/takeover-2016.js

```diff
@@ -37,13 +37,16 @@
         });
     }
 
-    // sets the mozorg.takeover state to true indicating that the user has
-    // seen the takeover modal during their current session.
-    function storeSession() {
+    // Sets a cookie so you won't see the takeover again for seven days
+    function seenTakover() {
+        var date = new Date();
+        date.setTime(date.getTime() + (7 * 24 * 60 * 60 * 1000));
+        var expires = date.toGMTString();
```

Comment:
According to MDN, [`toGMTString()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toGMTString) is deprecated. I guess [`toUTCString()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toUTCString) is the way to go?